src/solver/type_getters.jl

```diff
@@ -600,12 +600,19 @@ function get_sim_info(config::AtmosConfig)
     isnothing(restart_file) ||
         @info "Restarting simulation from file $restart_file"
     epoch = DateTime(parsed_args["start_date"], dateformat"yyyymmdd")
+    t_start_int = time_to_seconds(parsed_args["t_start"])
+    if !isnothing(restart_file) && t_start_int != 0
+        @warn "Non zero `t_start` passed with a restarting simulation. The provided `t_start` will be ignored."
```

Comment:
I thought this PR was enabling using non-zero `t_start` in a restarted simulation, but this suggests it's still not supported

Reply:
It enables non-zero `t_start` in a _manually_ restarted simulation (as we do in the coupler). If you restart from the Atmos checkpoints, `t_start` is already correctly set (because it is read from file), so there's no need to specify a different one.

Reply:
Added some additional documntation about this